cli/src/commands/upload.command.ts

```diff
@@ -276,12 +276,12 @@ export class UploadCommand extends BaseCommand {
         'x-api-key': this.immichApi.apiKey,
         ...data.getHeaders(),
       },
-      maxContentLength: Infinity,
-      maxBodyLength: Infinity,
+      maxContentLength: Number.POSITIVE_INFINITY,
+      maxBodyLength: Number.POSITIVE_INFINITY,
       data,
     };
 
-    const res = await axios(config);
-    return res;
+    const response = await axios(config);
+    return response;
```

Comment:
```suggestion
    return await axios(config);
```

Reply:
Thanks Ben. I agree with this, will try to find a decent balance with which unicorn rules we apply.